etc/helm/pachyderm/templates/pachd/rbac/role.yaml

```diff
@@ -53,6 +53,8 @@ rules:
   resources:
   - deployments
   - deployments/scale
+  - replicasets
```

Comment:
Is this still needed after the move to deployments?

Reply:
no its not, good catch